docs/release-notes/NuGet-5.7.md

```diff
@@ -0,0 +1,94 @@
+
+* Improve DOTNET_HOST_PATH environment variable handling - [#7438](https://github.com/NuGet/Home/issues/7438)
+
+* nuget.exe spec generates a copyright tag with hard-coded text of Copyright YYYY Instead of $copyright$ - [#8696](https://github.com/NuGet/Home/issues/8696)
+
+* nuget cli 3.5.0 throws exception 'authors required' during pack of a csproj ignoring placeholders and assemblyinfo attributes if the assembly name is changed - [#4234](https://github.com/NuGet/Home/issues/4234)
+
+* Consider updating the libraries to use newer versions of Newtonsoft.Json - [#9263](https://github.com/NuGet/Home/issues/9263)
+
+* HttpSourceAuthenticationHandler SemaphoreSlim future proofing - [#9463](https://github.com/NuGet/Home/issues/9463)
+
+* HttpRequestMessage gets reused multiple times which is not supported with the SocketHttpHandler - [#8661](https://github.com/NuGet/Home/issues/8661)
+
+* NuGet.Indexing 5.6.0 preview 3 and later use a different public key token - [#9481](https://github.com/NuGet/Home/issues/9481)
+
+* Honor TreatWarningsAsErrors during NuGet Package creation - [#7404](https://github.com/NuGet/Home/issues/7404)
+
+* [CPVM] Spurious package downgrades for multiple p2p projects  - [#9549](https://github.com/NuGet/Home/issues/9549)
+
+* The “Browse” tab is not aligned left with search box - [#9559](https://github.com/NuGet/Home/issues/9559)
+
+* The installed version is inconsistent with the embedded icon in the solution level PM UI for one package id with multiple versions installed - [#9321](https://github.com/NuGet/Home/issues/9321)
+
+* Leak: PartCreationPolicy(CreationPolicy.NonShared) NuGet.SolutionRestoreManager.RestoreOperationLogger - [#9595](https://github.com/NuGet/Home/issues/9595)
+
+* Avoid reading the assets file in no-op restores - [#9693](https://github.com/NuGet/Home/issues/9693)
+
+* Current Source Link package is not working - [#9695](https://github.com/NuGet/Home/issues/9695)
+
+* NuGet.Protocol does not support getting a version's download count from search - [#9086](https://github.com/NuGet/Home/issues/9086)
+
+* Improve memory performance of PackageMetadataResourceV3 by reducing the JObject dependencies - [#9719](https://github.com/NuGet/Home/issues/9719)
+
+**DCRs:**
+
+* Suppressed the <owners> element when it is redundant - [#5134](https://github.com/NuGet/Home/issues/5134)
+
+* Log IntervalTrackers as ETW events - [#9593](https://github.com/NuGet/Home/issues/9593)
+
+* Added an informational message on restore to inform CPVM users that the feature is in preview - [#9340](https://github.com/NuGet/Home/issues/9340)
+
+* Populate Solution Explorer package/project transitive dependencies from assets file - [#9580](https://github.com/NuGet/Home/issues/9580)
+
+* Visual Studio partial restore for PackageReference projects (noop++) prototype - [#9513](https://github.com/NuGet/Home/issues/9513)
```

Comment:
>prototype [](start = 71, length = 9)

We should remove "prototype" since we delivered the feature.

Reply:
Also, I think we should move this to the features section.

---
In reply to: [474354607](https://github.com/NuGet/docs.microsoft.com-nuget/pull/2137#discussion_r474354607) [](ancestors = 474354607)